src/V3Timing.cpp

```diff
@@ -550,7 +551,9 @@ class TimingControlVisitor final : public VNVisitor {
             ss << '"';
             V3EmitV::verilogForTree(sensesp, ss);
             ss << '"';
-            auto* const commentp = new AstCExpr{sensesp->fileline(), ss.str(), 0};
+            // possibly a multiline string
+            const std::string comment = std::regex_replace(ss.str(), std::regex("\n"), "\"\n\"");
+            auto* const commentp = new AstCExpr{sensesp->fileline(), comment, 0};
```

Comment:
What is making the multiline comments?  Not obvious what changed here to need this?

Reply:
Verilator emits verilog code that is represented by `sensesp` and puts it as a comment in the generated c++. I don't know if it is really helpful for a user, especially that this code is not always a correct verilog. But if it already is, I decided to keep it.
The verilog code of AstExprStmt consists of a few lines:
https://github.com/verilator/verilator/blob/2daa32b98b27855a224ce4daea5a5b6702004af2/src/V3EmitV.cpp#L428-L431
So I needed to handle the case with multiline comments.

Reply:
Ok, I suggest remove the code that makes them.  I don't like using regexp for simple replaces as is slow.... but we can thus avoid entirely.

Reply:
Ok, done.

Reply:
I was wrong. It is not a comment, but a function argument. Its value doesn't change the runtime behavior. It is rather for debug purposes.

Reply:
These arguments (`eventDescription`) are used only in debug messages:
https://github.com/verilator/verilator/blob/63db60f64618798796482595e8174544d00cf5dd/include/verilated_timing.h#L320-L340
I think it's better to keep them. I replaced `regex` with simple `remove`, because we only remove `\n` and no regex is needed.